WcaOnRails/app/models/team.rb

```diff
@@ -75,7 +75,14 @@ def self.all_officers
 
   # Code duplication from Cachable concern, as we index by friendly_id and not by id :(
   def self.c_all_by_friendly_id
-    @@teams_by_friendly_id ||= all.with_hidden.index_by(&:friendly_id)
+    @@teams_by_friendly_id ||= nil
```

Comment:
Why are you setting `teams_by_friendly_id` explicitly to `nil`, but not `teams_by_friendly_id_timestamp`?
Please change it for either both or none. I think unknown variables in Ruby are set to `nil` anyways, so my preference would be to drop this line altogether. But please check about the implicit `nil` beforehand.

Reply:
I'm pretty sure we need to initialize a class variable before accessing it. For `teams_by_friendly_id_timestamp`teams_by_friendly_id_timestamp it's not the case, because it's guarded by the `@@teams_by_friendly_id.nil?` check. That said, initializing both explicitly sounds good too :)

Reply:
I've initialized `teams_by_friendly_id_timestamp` as well to nil.